src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+@import "./card.scss";
+@import "./stars.scss";
+@import "../utils/variables";
+
+* {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
only `box-sizing: border-box;` we use for `*` selector. Move other rules to html/body tag selector